src/store/modules/user/actions.ts

```diff
@@ -93,6 +93,7 @@ const actions: ActionTree<UserState, RootState> = {
         await dispatch('getEComStores', { facilityId: resp.data.facilities[0].facilityId })
       }
       commit(types.USER_CURRENT_FACILITY_UPDATED, resp.data.facilities.length > 0 ? resp.data.facilities[0] : {});
+      dispatch('getFacilityLocations', resp.data.facilities.length > 0 ? resp.data.facilities[0].facilityId : '')
```

Comment:
We always need to fetch locations for 0th facility, for now add a TODO to remove this check once adding support to not allow user login without facilities.